### DISCUSSION TITLE
Welcome to FS-AI_ADS-DV_CAD Discussions!

### DISCUSSION THREAD

**Discussion by IanMurphy-Rockfort** [FS-AI/FS-AI_ADS-DV_CAD#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RoniEmad** [comment]
Hi Ian,

Regarding the additional sensors, as an international team, is it possible for us to have our sensor kit reviewed before the competition? We need to ensure that our kit meets all the necessary requirements and regulations prior to our travel.
It would be greatly appreciated if you can assist in providing guidance or referring us to a contact person who can help us navigate the review process.

Additionally, i would like to inquire if it is acceptable to make some changes to the submitted kit ? we would like to take into account factors such as ease of accessibility and weather changes.

Also, regarding the permitted envelope, is there any tolerance for deviations? For example, if a sensor is positioned slightly outside the specified limits, as shown in the picture, by approximately 5cm, would that be acceptable?

Thank you for your assistance, Roni

![image](https://github.com/FS-AI/FS-AI_ADS-DV_CAD/assets/90726457/101a5431-1caa-438b-aa1e-aefefd446621)

**Comment by IanMurphy-StreetDrone** [comment]
> Regarding the additional sensors, as an international team, is it possible for us to have our sensor kit reviewed before the competition? We need to ensure that our kit meets all the necessary requirements and regulations prior to our travel.
> It would be greatly appreciated if you can assist in providing guidance or referring us to a contact person who can help us navigate the review process.

The CAD you submitted has been reviewed and there are two issues.

1.  The 35mm overlap you show in the diagram.
2.  No details of wiring strain relief appear in the CAD

The first opportunity for physical review will be at the Static Integration event on Tues 18th July, which I believe your team is registered for.

> Additionally, i would like to inquire if it is acceptable to make some changes to the submitted kit ? we would like to take into account factors such as ease of accessibility and weather changes.

Yes, it's OK to change the specification but you will need to inform the Event Captain for each event of any changes you make. The Event Captain will decide if you need to present any further evidence and / or if the changes are acceptable.

> Also, regarding the permitted envelope, is there any tolerance for deviations? For example, if a sensor is positioned slightly outside the specified limits, as shown in the picture, by approximately 5cm, would that be acceptable?

All sensors need to fit within the limits in the CAD model. So the overlap shown in the diagram is not compliant. When on the real car, the position of the wheels defining the envelope is inconsistent, so teams will not be penalised for a small deviation. But the nominal design location within the CAD model must be compliant.

Thanks, Ian

**Comment by RoniEmad** [comment]
Thanks for reply.
> The CAD you submitted has been reviewed and there are two issues.
> 
> 1. The 35mm overlap you show in the diagram.
> 2. No details of wiring strain relief appear in the CAD

Okay, I will edit these issues.

> Yes, it's OK to change the specification but you will need to inform the Event Captain for each event of any changes you make. The Event Captain will decide if you need to present any further evidence and / or if the changes are acceptable.

Okay, This will be physically in the competition, right ?

> All sensors need to fit within the limits in the CAD model. So the overlap shown in the diagram is not compliant. When on the real car, the position of the wheels defining the envelope is inconsistent, so teams will not be penalised for a small deviation. But the nominal design location within the CAD model must be compliant.

Okay, I understand.
How will I re-submit the new CAD model ? on the website or via email ?

Thanks, Roni

**Comment by IanMurphy-Rockfort** [comment]
> Yes, it's OK to change the specification but you will need to inform the Event Captain for each event of any changes you make. The Event Captain will decide if you need to present any further evidence and / or if the changes are acceptable.
> 
> Okay, This will be physically in the competition, right ?

This will be at each of the Dynamic Events, so if for example you decide to change something after the Acceleration, for the Skid Pad, you need to inform the Event Captain at the Skid Pad dynamic event.
> How will I re-submit the new CAD model ? on the website or via email ?

If the website still accepts uploads, use that Otherwise, at the at this late stage, you can provide a download link for the CAD data by email to the IMechE. You should also bring your CAD files to the competition.

Thanks, Ian

**Reply by RoniEmad** [reply]
Okay, Thank you so much.

**Comment by Rikimadi** [comment]
Hi Ian, 

I'm also an international team member and I read the previous comment.
First it seems like we didn't have any bad or good review for our ASF, is it normal?

Then my team and I are wondering: What is the pinout of the external electrical power connector for our sensors?
Is GND on pin 1 and 12V on pin 2 or is it 12V on pin 1 and GND on pin 2 please?

Thank you for your time and have a nice day, 
Rikinaze

**Comment by IanMurphy-StreetDrone** [comment]
> First it seems like we didn't have any bad or good review for our ASF, is it normal?

This is down to availability of volunteer resource. Please be reassured we will support you at the event to ensure your sensors are compliant and will function correctly.

> Then my team and I are wondering: What is the pinout of the external electrical power connector for our sensors?
Is GND on pin 1 and 12V on pin 2 or is it 12V on pin 1 and GND on pin 2 please?

I have updated the info with the pinout: Pin 1 = 12V and pin 2 = Gnd.

Thanks, Ian

**Reply by Rikimadi** [reply]
Okay thank you very much for these informations ! 

Rikinaze

**Comment by fruffers** [comment]
Hi Ian, 

I hope you are well, 

are you able to tell us how much is the weight of the ADS-DV in terms of its body and battery/components? Sorry I have not been able to find this information. 

Warm regards,
Louise.

**Comment by IanMurphy-StreetDrone** [comment]
Hi Louise,
The total mass of the ADS-DV is approx. 300kg. We don't have a breakdown of the various component masses.
Thanks, Ian

**Reply by fruffers** [reply]
Hi Ian, thanks so much! :)

Would you also happen to know these measurements for the ADS-DV: Steering ratio, turning radius, pivot points (points where front wheels are attached to the suspension)? 

Kind regards.

**Reply by IanMurphy-StreetDrone** [reply]
The CAD model includes the steering and suspension modelled in a simplified form with spheres at all pivot points.

**Reply by fruffers** [reply]
Okay. Thank you a lot Ian! :)

**Reply by fruffers** [reply]
Hello Ian, 

Would you happen to know how many teeth are on the ADS-DV's wheelspeed sensors? I am trying to simulate the wheelcounts (pulses) of the ADS-DV and I'm not sure how to do it without knowing this value. 

Kind regards.

**Reply by IanMurphy-StreetDrone** [reply]
According to the CAD the wheelspeed triggers have 20 teeth. I am unable to verify this for sure without access to a vehicle.

Thanks, Ian

**Reply by fruffers** [reply]
Thank you Ian!

I would also like to ask when commands are recieved by the VCU of the ADS-DV does it already have inside it a PID controller or this is something that has to be implemented in the software of the students? Sorry if I am not making sense, I'm a bit confused about it. 

Kind regards,
Louise.

**Reply by IanMurphy-StreetDrone** [reply]
This is the CAD repository. The VCU interface is documented here:

https://github.com/FS-AI/FS-AI_API

**Reply by fruffers** [reply]
Thank you Ian! Do you also know the mass of the wheels? Also do you know whether there is a maximum speed the ADS-DV can go? Sorry if this is the wrong forum to ask.

**Reply by IanMurphy-StreetDrone** [reply]
A detailed mass breakdown for the ADS-DV is not available.

Please review all info at: https://github.com/FS-AI/FS-AI_API

Maximum motor speed is 4000rpm, with enough distance it may be possible to reach this speed...

**Reply by fruffers** [reply]
Thank you very much again Ian! :)

**Comment by Mohamed-alaa32** [comment]
Hi Ian,

I am writing to inquire about the submission process for the CADs related to the additional sensors for the DDT class.

Upon reviewing the Documents submissions section, I noticed that there is no designated submission slot for these CAD files. 

Could you please advise on where we should submit the CAD files for the additional sensors? As the deadline of submitting these CAD files is after 3 days so if there is a specific process or location for such submissions that I might have missed, I would greatly appreciate your guidance in this matter.


Additionally, if there are any specific formatting requirements or additional information needed along with the CAD files, please let me know so that I can ensure a smooth and efficient submission process.


Thank you very much for your attention to this matter. I look forward to your response and guidance on how to proceed with the submission.

**Comment by IanMurphy-StreetDrone** [comment]
Document submission queries need to be referred to the Formula Student office.

Format requirements are clearly specified here: https://github.com/FS-AI/FS-AI_ADS-DV_CAD?tab=readme-ov-file#additional-sensor-and-compute-mounting-cad-submission-for-approval

Thanks, Ian

**Reply by Mohamed-alaa32** [reply]
Thank you so much.

**Comment by marc-antoine** [comment]
Hi Ian,

I'm writing to enquire about the CAD submission process for the additional sensors for the DDT class.

We couldn't find the CAD section on the document submission page (I've attached a screenshot to this page). I read in the readme of the link you sent that our team leader should have received an email. But we've checked all our inboxes (inbox, spam...) several times but there's nothing. However, we did request an additional sensor in the "DESIGN CONCEPT SPECIFICATION".
We are therefore unable to submit our CAD work. Could you please tell us how to proceed?

Thank you very much for your attention to this question. 

Marc-Antoine 
Head of Competition ARECE ECE Paris

<img width="1137" alt="Submission page" src="https://github.com/FS-AI/FS-AI_ADS-DV_CAD/assets/84786796/e380bfbf-056a-400e-a3f8-a7fba5e19aba">

**Reply by IanMurphy-StreetDrone** [reply]
Document submission queries need to be referred to the Formula Student office.

**Comment by henrichhegedus** [comment]
Hi Ian,

Exciting to hear there are going to be two more ADS-DVs.
I have a few questions about the updates to the cars.

- Is the rear motor also being upgraded or will the car have effectively half the power?
- Is the new GPU the 3GB or 6GB variant of the 1060?

Thank you in advance,
Henrich

**Reply by IanMurphy-StreetDrone** [reply]
Hi,

The rear motor is the same as before so the vehicle's acceleration will be reduced.

The GPU RAM is not mentioned anywhere on the specification sheet: https://www.in-carpc.co.uk/wp-content/uploads/2021/01/CQ70-Datasheet.pdf so I cannot answer that query.

Thanks, Ian

**Reply by henrichhegedus** [reply]
Thank you for the quick response Ian.
From my understanding the ADS-DVs that are being borrowed for testing purposes haven't been upgraded yet. Is that correct?

If so, is it safe to send 0 torque to the front axle while driving the car with the rear only in order to simulate how the car is going to behave with only one motor?

**Reply by IanMurphy-StreetDrone** [reply]
Correct, the existing cars have not been changed yet.

The API functions don't allow for sending different torque request to the front and rear motors. If you modify them, then yes you could send torque request only to the rear.

Thanks, Ian
